src/js/util/slide.js

```diff
@@ -0,0 +1,39 @@
+export default class Slide {
+  constructor(el, collapsedClass = 'collapsed') {
```

Comment:
Not sure if there's any intent to expose this class for use outside this library, but it might be smart to use an object for these options instead of params. 

Makes things a bit more flexible for future adjustments.

Reply:
good call